### PR TITLE
feat: add changelog action

### PR DESCRIPTION
**Pull request by klmcadams**
## Description
Add the changelog action to pyansys-geometry

## Issue linked
**Please mention the issue number or describe the problem this pull request addresses.**

## Checklist
- [x] I have tested my changes locally.
- [x] I have added necessary documentation or updated existing documentation.
- [x] I have followed the coding style guidelines of this project.
- [x] I have added appropriate unit tests.
- [x] I have reviewed my changes before submitting this pull request.
- [x] I have linked the issue or issues that are solved to the PR if any.
- [x] I have assigned this PR to myself.
- [x] I have made sure that the title of my PR follows [Conventional commits style](https://www.conventionalcommits.org/en/v1.0.0/#summary) (e.g. ``feat: extrude circle to cylinder``)

### CODE REVIEW

File: CHANGELOG.md
Comment (PipKat) [inline CHANGELOG.md:4]:
```suggestion
This project uses [towncrier](https://towncrier.readthedocs.io/) to generate changelogs. You can see the changes for the upcoming release in <https://github.com/ansys/{repo-name}/tree/main/changelog.d/>.

```
We let the link be enough emphasis for a new product, so I removed the italics.

**Review by PipKat — APPROVED** [review]
Minor comments on doc-related changes.

File: .github/workflows/label.yml
Comment (RobPasMue) [inline .github/workflows/label.yml:99]:
```suggestion
    - uses: ansys/actions/doc-changelog@main
```

We have to use main - just letting this comment for whenever the action is merged.

File: pyproject.toml
Comment (RobPasMue) [inline pyproject.toml:163]:
```suggestion
showcontent = true

```

EOF line missing

File: doc/changelog.d/changelog_template.jinja
Comment (RobPasMue) [inline doc/changelog.d/changelog_template.jinja:15]:
```suggestion
{% endif %}

```
EOF line missing

File: .github/workflows/label.yml
Comment (RobPasMue) [inline .github/workflows/label.yml:101]:
```suggestion
        token: ${{ secrets.PYANSYS_CI_BOT_TOKEN }}

```
EOF line missing

**Review by RobPasMue — CHANGES_REQUESTED** [review]
It's looking good =) Thanks @klmcadams - I will "request changes" in order to block this PR from merging until the action is merged. Once merged, I will approve.

**Review by RobPasMue — APPROVED** [review]
LGTM! Let's leave it with ``main`` for now. Once v6 is out I'll do the change. This will also allow you to play with the conventional commits approach =)

**Pull request merged**
